boards/arm/lora_e5_dev_board/lora_e5_dev_board.dts

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) 2021 Thomas Stranger
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/wl/stm32wle5Xc.dtsi>
```

Comment:
Should we have a dtsi for the module? Similar to the one in https://github.com/zephyrproject-rtos/zephyr/tree/main/dts/arm/acsip.

The LoRa-E5 includes the MCU, TCXO, low frequency crystal, and radio switch (https://www.disk91.com/2021/technology/lora/lora-e5-the-seeed-studio-stm32-wl-module/). Thinking it should have the` stm32wle5Xc.dtsi` include, the clock nodes (`rcc`, `pll` and `clk_msi`) and then I can rebase the radio patch and add the radio node there with the RF switch pin mapping.

Reply:
I have added a nearly empty lora-e5.dtsi in the arm/seeed folder.
The external clock frequencies are already defined in the soc dtsi, therefore I have not defined them here again.
I have left out further clock configuration(rcc, pll) as this may often be adjusted by the user per board.

Please let me know what you think about introducing a "lora-e5" module and if you would add further definitions.

Reply:
Yeah perfect! I'll rebase the lora patch on top and add the subghz node with the module setting there (like the acsip one) once this is merged. Thanks!